src/state/wallets/selectors.ts

```diff
@@ -146,7 +146,7 @@ export const transactions = createSelector(wallets, electrumXSelectors.blockHeig
       const id = wallet.id;
       return wallet.transactions.map(transaction => {
         const { height } = transaction;
-        const confirmations = max([height > 0 ? blockHeight - height : 0, 0]) || 0;
+        const confirmations = max([height > 0 ? blockHeight + 1 - height : 0, 0]) || 0;
```

Comment:
Maybe let's leave a comment to know why we add this `1` here?

Reply:
done